multimedia/graphicsmagick/Makefile

```diff
@@ -31,7 +31,7 @@ define Package/graphicsmagick/Default
   SECTION:=multimedia
   CATEGORY:=Multimedia
   TITLE:=Graphics manipulation tools
-  URL:=https://www.graphicsmagick.org/
+  URL:=http://www.graphicsmagick.org/
```

Comment:
Have you reached upstream developers regarding their HTTPS problem? I've briefly checked their website: http://www.graphicsmagick.org/ and I can see there this:

> Check http://www.graphicsmagick.org/ or https://graphicsmagick.sourceforge.io/index.html for the latest version of this page.

Reply:
You keep making up non-existent problems all over the Internet and wanting me to solve them for you. This is not gonna happen.